R/comparison.R

```diff
@@ -0,0 +1,93 @@
+rcmdcheck_comparison <- function(old, new) {
+  stopifnot(is.list(old))
+  stopifnot(inherits(new, "rcmdcheck"))
+
+  # Generate data frame of problems
+  old_df <- do.call(rbind, lapply(old, as.data.frame, which = "old"))
```

Comment:
I think it is a nice feature, and it is easy to implement, just download and parse with the same parser.